README.md

```diff
@@ -6,6 +6,17 @@ The infrastructure consists of the following components:
 
 ## Setup infrastructure
 Copy `.env-template` to `.env` and fill with the appropriate values.  
+
+### Workflow configuration
+The available workflows are configured via a json file, for an example see: `/config/workflow_config_example.json`  
+A workflow must have a `workflow_type_name` and `workflow_type_description_name`.
+The `workflow_parameters` are optional, for workflows that need additional information next to the ESDL.
+Each workflow parameter must have a `parameter_type` and `key_name`, all others are optional.  
+See `/config/workflow_config_example.json` for options for the `string`, `boolean`, `integer`, `float` and `datetime` parameter formats.
```

Comment:
```suggestion
See `./config/workflow_config_example.json` for options for the `string`, `boolean`, `integer`, `float` and `datetime` parameter formats.
```
`/` is reserved in Linux as the 'root' of the file system. As far as I know there isn't a symbol to denote the root of the repository, but otherwise I would think Linux users will get confused if we use `/`. We could use `./`, or perhaps `<repo>/` or something similar?

Reply:
I opted for leaving out the `/`: fcdd4978d32f86633f6e9499ea360ea4b450bede